Week3/homework/2-step3.js

```diff
@@ -1,8 +1,37 @@
-'use strict';
+"use strict";
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const values = [];
-  // Add your code here
+  const divisibleBy3 = [];
+  const divisibleBy5 = [];
+
+  if (startIndex > stopIndex) {
+    stopIndex = [startIndex, (startIndex = stopIndex)][0];
+  }
+
```

Comment:
The idea here is nice, if I understand it correctly.
You are switching the start and stop indexes in case the start is lower than the stop, correct?

Your way of doing it here is pretty confusing, and I had to test it to see if it would actually work.  Kudos that it actually does.  But it is very hard to figure out what is happening here.  You are assigning a new value by putting the other value in an array, and then assigning a new value to the startIndex in the same array, and then taking that first value of the array. 
I can see why you are doing this, to hold your values temporarily, but I would *emphatically* suggest not doing this in the real world. 😉 
I can't think of a way to get around using a temporary variable, that is clean.
```
if (startIndex > stopIndex) {
  let tempStop = stopIndex;
  stopIndex = startIndex;
  startIndex = stopIndex
}
```
It ain't sexy, but it gets the job done and is still readable.
**But** extra credit for the very creative solution.  I kind of love it, even though I would never dare use it in the wild. 😆